### PR TITLE
Cleaned up the mkdocs.yml file.

### PR DESCRIPTION
**Pull request by therealmrmuffin**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,15 +1,13 @@
 site_name: SimpleLogin Docs
-site_url: https://simplelogin.io/docs/
+site_url: https://simplelogin.io/
```
Comment (nguyenkims) [inline mkdocs.yml:2]: The `site_url` actually corresponds to where the doc is hosted which is https://simplelogin.io/docs/ and not https://simplelogin.io

File: mkdocs.yml [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
 site_description: Receive and send emails anonymously.
 site_author: SimpleLogin
 
 # Repository
 repo_name: simple-login/documentation
 repo_url: https://github.com/simple-login/documentation
-copyright: SimpleLogin is an open-source email alias solution to protect your email address from spam and more.
```
Comment (nguyenkims) [inline mkdocs.yml:9]: the `copyright` here is indeed a bit misleading, it's rather used as the site footer and not like as an actual copyright section.

Comment (therealmrmuffin) [reply mkdocs.yml:9]: Okay!.

File: mkdocs.yml (hunk 1 above)
Comment (therealmrmuffin) [reply mkdocs.yml:2]: Ho! didn't know that. Trying to make the image in the top left of the docs link to the main site.

**Pull request closed without merging**